qcodes/instrument/channel.py

```diff
@@ -732,33 +734,33 @@ def remove(self)->None:
             self._channel_list.remove(self)
         self._exists_on_instrument = False
 
-    def _remove(self) ->None:
+    def _remove(self) -> None:
         """
         (SCPI) commands needed to delete the channel from the instrument
         """
         raise NotImplementedError("Please subclass")
 
-    def _assert_existence(self)->None:
+    def _assert_existence(self) -> None:
         if not self._exists_on_instrument:
             raise RuntimeError(
                 "Object does not exist (anymore) on the instrument")
 
-    def write(self, cmd: str)->Any:
+    def write(self, cmd: str):
```

Comment:
Why not ` -> None` for this one? Is `mypy` unhappy about the `super()`?

Reply:
Because @astafan8 is going to tell me that its redundant :)